js/renovation/viz/core/__tests__/tooltip.test.tsx

```diff
@@ -90,24 +166,124 @@ describe('Render', () => {
   });
 
   it('should render text with props', () => {
-    const props = {
-      size: {
-        width: 40, height: 30, x: 1, y: 2,
+    const tooltip = shallow(<TooltipComponent {...props as any} /> as any);
+
+    expect(tooltip.find('TextSvgElement').props()).toMatchObject({
+      text: 'customized_text',
+      styles: {
+        fill: 'customized_font_color',
+        fontFamily: 'test_family_color',
+        opacity: 0.4,
+        fontSize: 15,
+        fontWeight: 600,
       },
-      textRef: {},
-      props: tooltipProps,
-    } as Partial<Tooltip>;
+    });
+  });
+
+  it('should render shadow', () => {
     const tooltip = shallow(<TooltipComponent {...props as any} /> as any);
 
+    expect(tooltip.find('defs')).toHaveLength(1);
+    expect(tooltip.find('ShadowFilter').props()).toMatchObject({
+      id: 'filterId',
+      x: '-50%',
+      y: '-50%',
+      width: '200%',
+      height: '200%',
+      blur: 2,
+      color: '#000',
+      offsetX: 0,
+      offsetY: 4,
+      opacity: 0.4,
+    });
+    expect(tooltip.find('g').at(0).props()).toMatchObject({
+      filter: 'url(#filterId)',
+    });
+
+    expect(getFuncIri).toBeCalledWith('filterId');
+  });
+
+  it('should render div for html text', () => {
+    const customizedOptions = { ...props.customizedOptions, html: 'html text' };
+    const tooltip = shallow(
+      <TooltipComponent {...{ ...props, customizedOptions } as any} /> as any,
+    );
```

Comment:
```suggestion
    const tooltip = shallow(viewFunction({ ...props, customizedOptions }));
```

This way is simpler.